systemd/ocp-custom-domain.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=CRC Unit setting nip.io domain for cluster
+After=kubelet.service
+Requires=kubelet.service
+
+[Service]
+Type=oneshot
+EnvironmentFile=/opt/crc/crc-cloud
```

Comment:
This service fails for crc because this environment file is not available so may be we should touch that file or may be check if file exist then run this service otherwise not.

Reply:
the cluster will still start and it doesn't have any adverse effect even if this service fails, so my initial thought was that this service failing to start is a good indication that the bundle is not running in crc-cloud